code/frameworks/nextjs/src/images/next-image-stub.tsx

```diff
@@ -38,7 +39,11 @@ const OriginalNextImage = NextImage.default;
 Object.defineProperty(NextImage, 'default', {
   configurable: true,
   value: (props: _NextImage.ImageProps) => {
-    return <OriginalNextImage {...props} loader={props.loader ?? defaultLoader} />;
+    const imageParameters = React.useContext(ImageContext);
+
+    return (
+      <OriginalNextImage {...props} loader={props.loader ?? defaultLoader} {...imageParameters} />
```

Comment:
We want passed parameters to override any other props, which is why we spread `imageParameters` last.

Reply:
As a user I would rather assume, that `imageParameters` are the defaults and I am still able to override the default case by case if I want.

```
<OriginalNextImage {...imageParameters} {...props} loader={props.loader ?? defaultLoader} />
```

Reply:
I think you're right. Treating `imageParameters` as the defaults makes sense to me!